Crash

### Prerequisites

- [X] I have written a descriptive issue title
- [X] I have searched existing issues to ensure it has not already been reported


### Fastify version

not fastify

### Plugin version

_No response_

### Node.js version

14

### Operating system

Linux

### Operating system version (i.e. 20.04, 11.3, 10)

20.04

### Description

```
internal/modules/cjs/loader.js:905
  throw err;
  ^

Error: Cannot find module 'pre-commit'
Require stack:
- /home/matteo/repositories/on-exit-or-gc/[eval]
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:902:15)
    at Function.resolve (internal/modules/cjs/helpers.js:98:19)
    at [eval]:1:21
    at Script.runInThisContext (vm.js:134:12)
    at Object.runInThisContext (vm.js:311:38)
    at internal/process/execution.js:77:19
    at [eval]-wrapper:6:22
    at evalScript (internal/process/execution.js:76:60)
    at internal/main/eval_string.js:23:3 {
  code: 'MODULE_NOT_FOUND',
  requireStack: [ '/home/matteo/repositories/on-exit-or-gc/[eval]' ]
}
internal/modules/cjs/loader.js:905
  throw err;
  ^

Error: Cannot find module 'pre-commit'
Require stack:
- /home/matteo/repositories/on-exit-or-gc/[eval]
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:902:15)
    at Function.resolve (internal/modules/cjs/helpers.js:98:19)
    at [eval]:1:21
    at Script.runInThisContext (vm.js:134:12)
    at Object.runInThisContext (vm.js:311:38)
    at internal/process/execution.js:77:19
    at [eval]-wrapper:6:22
    at evalScript (internal/process/execution.js:76:60)
    at internal/main/eval_string.js:23:3 {
  code: 'MODULE_NOT_FOUND',
  requireStack: [ '/home/matteo/repositories/on-exit-or-gc/[eval]' ]
}
```

### Steps to Reproduce

Install

### Expected Behavior

_No response_

--- a/hook
+++ b/hook
@@ -46,5 +46,5 @@ if [[ $* == *--dry-run* ]]; then
     exit 1
   fi
 else
-  "$BINARY" "$("$BINARY" -e "console.log(require.resolve('pre-commit'))")"
+  "$BINARY" "$("$BINARY" -e "console.log(require.resolve('@fastify/pre-commit'))")"
 fi

